doctrine/doctrine-bundle/2.8/manifest.json

```diff
@@ -11,7 +11,8 @@
         "#2": "IMPORTANT: You MUST configure your server version, either here or in config/packages/doctrine.yaml",
         "#3": "",
         "#4": "DATABASE_URL=\"sqlite:///%kernel.project_dir%/var/data.db\"",
-        "#5": "DATABASE_URL=\"mysql://app:!ChangeMe!@127.0.0.1:3306/app?serverVersion=8&charset=utf8mb4\"",
+        "#5": "DATABASE_URL=\"mysql://app:!ChangeMe!@127.0.0.1:3306/app?serverVersion=mariadb-10.11&charset=utf8mb4\"",
```

Comment:
You should specify the version as the server would report it, otherwise later versions of DBAL will fail to parse it.

```suggestion
        "#5": "DATABASE_URL=\"mysql://app:!ChangeMe!@127.0.0.1:3306/app?serverVersion=10.11.2-mariadb&charset=utf8mb4\"",
```

Reply:
I noticed that the DBAL docs are outdated in that regard.

Reply:
interesting, didn't know that.

Reply:
doctrine/dbal#5971